pycodestyle.py

```diff
@@ -1524,7 +1524,7 @@ def python_3000_invalid_escape_sequence(logical_line, tokens):
                     pos += 1
                     if string[pos] not in valid:
                         yield (
-                            pos,
+                            line.lstrip().find(text),
```

Comment:
Maybe I'm missing something, but wouldn't `lstrip` subtract the indentation from the index of `text`?

Reply:
In the following sections, `pos` is derived from `logical_line`. I followed this and I have removed indentation.
https://github.com/PyCQA/pycodestyle/blob/master/pycodestyle.py#L1438
https://github.com/PyCQA/pycodestyle/blob/master/pycodestyle.py#L1452

Reply:
Ah! that makes sense.  Thanks for explaining.